tests/acceptance/acceptance/Tainting.feature

```diff
@@ -1,4 +1,5 @@
 @symfony-common
+@tainting
```

Comment:
yep, I'm replacing it by `@symfony-common`

Reply:
this `@tainting` can be removed as well.

---------

On the other hand, I think it would be nice to add documentation about the feature and how to configure twigCachePath

> Thank you for great work! This one also needs a documentation update.

Reply:
done (removed the `@tainting` and updated the README)